website/src/components/discourse/index.js

```diff
@@ -34,41 +84,64 @@ export default function DiscourseFeed(styles = {}) {
     }
     fetchData()
   }, [])
+
+  // Set initial min-height
+  // This is to avoid layout shifts
+  // which affects Lighthouse performance scores
+  const setMinHeight = isError
+    ? 'auto'
+    : 414
+
   return (
-    <section className={feedStyles.discourseSection}>
-      <h1>dbt Discourse</h1>
-      <div className={feedStyles.discoursePosts}>
-        <h2>Latest posts</h2>
-        {loading ? (
-          <img src="img/loader-icon.svg" alt="Loading" className={feedStyles.loadingIcon} />
-        ) : isError || !posts?.length > 0 ? (
-          <p>Unable to load Discourse posts at this time.</p>
-        ) : (
-          <ul>
-            {posts.map(post => (
-              <li>
-                <PostWrapper post={post}>{post.title}</PostWrapper>
-                <span> {post.username && `- by ${post.username}`}</span>
-              </li>
-            ))}
-          </ul>
-        )}
-        
-       
-        <a className={`button button--primary ${feedStyles.discourseCta}`} href="https://discourse.getdbt.com/" title="See all latest posts" target="_blank">See all latest posts</a>
-      </div>
-    </section>
+    <div className={feedStyles.discourseTopics} style={{minHeight: setMinHeight, ...styles}}>
+      {title && (
+        <h2>{title}</h2>
+      )}
+      {loading ? (
+        <img src="/img/loader-icon.svg" alt="Loading" className={feedStyles.loadingIcon} />
+      ) : isError || !topics?.length > 0 ? (
+        <p>Unable to load forum topics at this time.</p>
+      ) : (
+        <ul>
+          {topics.map(topic => (
+            <li key={topic.id}>
+              {topic?.has_accepted_answer && (
+                <span className={feedStyles.solvedTopic} title="Solved">✅ </span>
+              )}
+              <TopicWrapper topic={topic}>{topic.title}</TopicWrapper>
```

Comment:
Also on the social proof front, it'd be nice to pull through the `like_count` from the posts array. I think this will have more purpose on the Discussion ones as I doubt people are going to click like on the OP of a support request, but if they do we can show them there too! 

I don't know how you render them - it might get messy to have `How do I do computers? by joellabes, 1 reply, 3 likes` but if we don't have design time then that might just have to do for now.

Reply:
Here's an example of how it looks with the like count added. Not many have the like count so not sure if any would show up in the preview, so adding it here. I had to set the `after` prop to `2022-01-01` on the first component within the /forum page to get it to show up.

`<DiscourseFeed title='Open topics' category='help' status='unsolved'  link_text='See open topics' link_href='https://discourse.getdbt.com/c/help/19' show_cta={true} after='2022-01-01' />`

![image](https://user-images.githubusercontent.com/13647110/191303169-18d2ffdb-436e-452d-b7a6-5183712e66fc.png)

Reply:
Works for me!